ena_upload/templates/ENA_template_experiments.xml

```diff
@@ -19,7 +29,9 @@
                         <PAIRED py:when="row.library_layout == 'paired'" NOMINAL_LENGTH="${row.insert_size}" />
                         <SINGLE py:when="row.library_layout == 'single'" />
                     </LIBRARY_LAYOUT>
+                    <py:if test="attributetest(row, 'library_construction_protocol')">
                     <LIBRARY_CONSTRUCTION_PROTOCOL>${row.library_construction_protocol}</LIBRARY_CONSTRUCTION_PROTOCOL>
+                    </py:if>
                 </LIBRARY_DESCRIPTOR>
             </DESIGN>
             <xi:include href="ENA_template_PLATFORM.xml" />
```

Comment:
Platform is composed of both manufacturer (called platform in the xml files) and instrument model.
In the [Webin-CLI](https://ena-docs.readthedocs.io/en/latest/submit/reads/webin-cli.html#manifest-file) it is mentioned that platform is unnecessary if an instrument model is provided.
In the webin submission portal the field "platform" is omitted and the user is presented with just `instrument_model`.
Can the same apply here? The user only provides `instrument_model` and the `platform` column of the `ENA_template_experiments.tsv` table is removed.

Reply:
is something we should discuss with @roncoronimiguel

Reply:
I think this will complicate things ...

Reply:
OK!
I think this can be left as it is now.